firestone/enUS.json

```diff
@@ -344,19 +344,7 @@
 					"battlegrounds": "Battlegrounds",
 					"mercenaries": "Mercenaries",
 					"book-of-mercs": "Book of Mercenaries",
-					"book-of-mercs-chapters": {
-						"rokara": "Rokara's story",
-						"xyrella": "Xyrella's story",
-						"guff": "Guff's story",
-						"kurtrus": "Kurtrus' story",
-						"tamsin": "Tamsin's story",
-						"cariel": "Cariel's story",
-						"scabbs": "Scabbs' story",
-						"tavish": "Tavish's story",
-						"brukan": "Bru'kan's story",
-						"dawngrasp": "Dawngrasp's story",
-						"other": "Other stories"
-					}
```

Comment:
If possible I would like to keep the existing translations until the new PR has been merged, to keep the backward compatibility.

Reply:
@piedpiper358 ^

Reply:
Done